function-aws-api-proxy/src/main/java/io/micronaut/function/aws/proxy/MapCollapseUtils.java

```diff
@@ -31,6 +35,8 @@
 @Internal
 public final class MapCollapseUtils {
 
+    private static final List<String> HEADERS_ALLOWING_COMMAS = Arrays.asList(HttpHeaders.DATE, HttpHeaders.USER_AGENT);
```

Comment:
Should this be configurable?

Reply:
I don't see how Netty handles this @yawkat do you know?

Reply:
> Should this be configurable?

but it should be configurable at not in AWS right?